cmd/kube-proxy/app/server.go

```diff
@@ -93,20 +97,45 @@ func (s *ProxyServer) Run(_ []string) error {
 	serviceConfig := config.NewServiceConfig()
 	endpointsConfig := config.NewEndpointsConfig()
 
-	protocol := iptables.ProtocolIpv4
-	if s.BindAddress.To4() == nil {
-		protocol = iptables.ProtocolIpv6
+	protocol := utiliptables.ProtocolIpv4
+	if net.IP(s.BindAddress).To4() == nil {
```

Comment:
It might have, building locally it passes and the e2e builds were passing until the last one, which I'm still reviewing (this may be the error source there).
This is clearly wrong regardless though, fixing.